package com.exception;

public class ex2 {
	public static void main(String[] args) {
		try {
			
			int i;
		int j;
		int result=10+10;
		System.out.println(result);
			
		} catch (Exception e) {
			int div=10/0;
			

		}

	}

}
